<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QWidget" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bgMainScreen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../LCS_CHECKWEIGHER/mainscreen.png</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtProductName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTargetLoad">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTareWeight">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtTotalCount">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtAcceptWeight">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtRejectWeight">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSettings">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>423</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
